csharp/extractor/Semmle.Extraction.CSharp.Standalone/AssemblyInfo.cs

```diff
@@ -139,6 +139,7 @@ public static AssemblyInfo ReadFromFile(string filename)
                  *  loading the same assembly from different locations.
                  */
                 using (var pereader = new System.Reflection.PortableExecutable.PEReader(new FileStream(filename, FileMode.Open, FileAccess.Read, FileShare.Read)))
+                using (var sha1 = new SHA1CryptoServiceProvider())
```

Comment:
Merge into one `using`?

Reply:
I would be inclined to keep them separate.